dynatrace/environment_v2/extensions.py

```diff
@@ -0,0 +1,162 @@
+        :param extension_name: the name of the extension 2.0
+
+        :return: a list of MinimalExtension objects
+        """
+        return PaginatedList(MinimalExtension, self.__http_client, target_url=f"{self.ENDPOINT}/{extension_name}", list_item="extensions")
+
+
+    def listEvents(self, extension_name: str) -> PaginatedList["ExtensionEventDTO"]:
+        """ List of the latest extension environment configuration events
+
+        :param extension_name: the name of the extension 2.0
+
+        :return: a list of ExtensionEventDTO object
+        """
+        return PaginatedList(ExtensionEventDTO, self.__http_client, target_url=f"{self.ENDPOINT}/{extension_name}/environmentConfiguration/events", list_item="extensionEvents")
+
+
+    def listEventsByConfigId(self, extension_name: str, config_id: str) -> PaginatedList["ExtensionEventDTO"]:
+        """ Gets the list of the events linked to specific monitoring configuration
+
+        :param extension_name: the name of the extension 2.0
+        :param config_id: The ID of the requested monitoring configuration.
+
+        :return: a list of ExtensionEventDTO object
+        """
+        return PaginatedList(ExtensionEventDTO, self.__http_client, target_url=f"{self.ENDPOINT}/{extension_name}/monitoringConfigurations/{config_id}/events", list_item="extensionEvents")
+        
+
+    def get(self, extension_name: str, extension_version: str) -> "Extension":
+        """ Gets details of specified version of the extension 2.0
+
+        :param extension_name: the name of the requested extension 2.0
+        :param extension_version: the version of the requested extension 2.0
+    
+        :return: a Extension class object
+        """
+        response = self.__http_client.make_request(f"{self.ENDPOINT}/{extension_name}/{extension_version}").json()
+        return Extension(raw_element=response)
+    
+
+    def delete(self, extension_name: str, extension_version: str):
+        """ Deletes the specified version of the extension 2.0
+
+        :param extension_name: the name of the requested extension 2.0
+        :param extension_version: the version of the requested extension 2.0
+
+        :return: HTTP response
+        """
+        return self.__http_client.make_request(path=f"{self.ENDPOINT}/{extension_name}/{extension_version}", method="DELETE")
+    
+
+    def getActiveExtensionVersion(self, extension_name: str) -> "ExtensionEnvironmentConfigurationVersion":
+        """ Gets the active environment configuration version of the specified extension 2.0
+
+        :param extension_name: the name of the requested extension 2.0
+
+        :return: ExtensionEnvironmentConfigurationVersion object
+        """
+        response = self.__http_client.make_request(f"{self.ENDPOINT}/{extension_name}/environmentConfiguration").json()
+        return ExtensionEnvironmentConfigurationVersion(raw_element=response)
+
+
+    def updateActiveExtensionVersion(self, extension_name: str, _version: str):
```

Comment:
`put_environment_config` This needs to be similar to other implementations like `SLOs`, where this method receives a `ExtensionEnvironmentConfigurationVersion` instance, and calls the `put()` method of that instance.